main.nf

```diff
@@ -38,6 +38,8 @@ def helpMessage() {
       --max_peptide_length [int]    Specifies the maximum peptide length Default: MHC class I: 11 aa, MHC class II: 16 aa 
       --min_peptide_length [int]    Specifies the minimum peptide length Default: MCH class I: 8 aa, MHC class II: 15 aa
       --tools [str]                 Specifies a list of tool(s) to use. Available are: 'syfpeithi', 'mhcflurry', 'mhcnuggets-class-1', 'mhcnuggets-class-2'. Can be combined in a list separated by comma.
+      --split_peptides_maxchunks [int]  Used in combination with '--peptides' or '--proteins': maximum number of peptide chunks that will be created for parallelization. Default: 100
```

Comment:
How about `--peptides_split_maxchunks`and `peptides_split_chunksize`?

Reply:
`split_peptides_minsize` is just the min. chunk size (apart for one file that contains the rest), it will become bigger if necessary to avoid more than `peptides_split_maxchunks` chunks .

`--peptides_split_minchunksize` or would that be too long?

Reply:
You mean the actual number of chunks that is created? That would be a third parameter, but yes it might be a good solution to keep it simple. Or do you really not want to change the `*-minchunksize` parameter, then we could let the user define `*-maxchunks`, I just understood that you wanted to change that as well ...

Reply:
I mean that it might be sufficient to have a `maxchunks`parameter since that defines the size of the chunks anyway indirectly.